testing/test_monkeypatch.py

```diff
@@ -327,3 +327,16 @@ def test_issue1338_name_resolving():
         monkeypatch.delattr('requests.sessions.Session.request')
     finally:
         monkeypatch.undo()
+
+
+def test_context(testdir):
+    testdir.makepyfile("""
```

Comment:
this can be constructed as unittest directly taking monkeypatch, no need to invoke a nested acceptance test

Reply:
Agreed. @feuillemorte please take a look at the other tests in this module for inspiration. 😁